source/feed.xml.builder

```diff
@@ -17,6 +17,8 @@ xml.feed "xmlns" => "http://www.w3.org/2005/Atom" do
       xml.published article.date.to_time.iso8601
       xml.updated article.date.to_time.iso8601
       #xml.author { xml.name "Article Author" }
+      image_url = article.image.url.gsub(%r{^//}, 'https://')
```

Comment:
In this case I would either use a helper to wrap image url transformation or added `article.image.url.gsub(%r{^//}, 'https://')` directly to  `xml.link` part. I don't really redefining variables inside iterators :nauseated_face: